src/data-workspace/data-entry-cell/entry-field-input.js

```diff
@@ -16,9 +16,10 @@ import { useDataValueParams } from './use-data-value-params.js'
 import { VALUE_TYPES } from './value-types.js'
 
 function createCurrentItem({ de, coc, dataValueSet }) {
-    if (dataValueSet?.data[de.id]?.[coc.id]) {
+    const dataValue = dataValueSet?.data.dataValues[de.id]?.[coc.id]
```

Comment:
this is the actual fix for date/name not being displayed correctly .. the data exists in `.dataValues` .. @Mohammer5 @HendrikThePendric please let me know if this change might have some unintended consequences

Reply:
This has been addressed already, but hasn't been merged: https://github.com/dhis2/data-entry-app/pull/73/files#diff-9ee92ce88ffa5ba79b5e4ef858a93314f19b500c331541899c6d368437ed3c74

I don't mind fixing it here either as it will make it into `development` much quicker than #73. The merge conflicts will be quite trivial to fix.